src/common/snippets/src/op/subgraph.cpp

```diff
@@ -433,7 +439,9 @@ void Subgraph::control_flow_transformations(lowered::LinearIR& linear_ir,
 
     lowered::pass::PassPipeline pipeline(lowered_pass_config);
     pipeline.register_pass<lowered::pass::MarkLoops>(vector_size);
+    // TODO: remove SoftmaxDecomposition pass
     pipeline.register_pass<lowered::pass::SoftmaxDecomposition>(vector_size);
```

Comment:
TODO:

1. Remove SoftmaxDecomposition
2. Update unit tests which are used this pass
3. Remove a workaround in PropagatePrecisions